app/src/Components/DisplayCards/DisplayCards.tsx

```diff
@@ -61,11 +79,15 @@ function DisplayCards ({ search, sortBy, order }: Props) {
           breakClassName={"break-btn"}
         />
       </div>
+      {isLoading && <h1>Loading...</h1>}
+      {!isLoading && (!cards || cards.length === 0) && <h1>No cards found</h1>}
       <div className="card-container">
```

Comment:
I like this. That way the user also knows if there are no cards. Before the user could have thought the app crashed.